src/scss/style.scss

```diff
@@ -482,6 +482,9 @@ button {
 .right-side {
   float: left;
   pointer-events: all;
+  p {
+    height: 22px;
+  }
```

Comment:
> Edit: also added fix for height issue I ran into on Chromium Edge (80.0.361.62 64-bit)

Looking at this CSS, it looks like you ran into a bug that we patched a couple days ago.

https://github.com/qmk/qmk_configurator/pull/666

If the [currently-deployed QMK Configurator](https://config.qmk.fm/) renders okay for you, the changes to this file should be reverted.